app/app.js

```diff
@@ -1,18 +1,16 @@
-import Ember from 'ember';
-import Resolver from './resolver';
+import Ember, { Application } from 'ember';
 import loadInitializers from 'ember-load-initializers';
-import config from './config/environment';
-
-let App;
+import { modulePrefix, podModulePrefix } from 'ember-get-config';
```

Comment:
Hmm. it's a bit cleaner.

The real test: It looks like ember-osf *always* makes sure this addon is installed, so in this case we're not bloating the app with an extra dependency just for semantics. Not sure I'd use it for a vanilla ember app, but ok here.